api/src/main/java/io/cloudevents/CloudEvent.java

```diff
@@ -24,9 +24,23 @@
  */
 public interface CloudEvent extends CloudEventAttributes, CloudEventExtensions {
 
+    /**
+     * The event data
+     */
+    @Nullable
+    Object getRawData();
```

Comment:
can we remove this one and replace with `getData(Object.class)`?

Reply:
Sounds good

Reply:
also, kinda the same applies to `geData(byte[].class)` / `getData(ByteBuffer.class)`, to avoid the duality of `getData()/getData(Class)` which may lead to questions like "should CloudEvent also handle the _serialization_ intto `byte[]` if it was created from an Object`